content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -449,14 +458,16 @@ Logout of the dashboard as admin and login the dashboard again as a non-admin us
 
 ### Multi Cloud Service
 
-1 Register object storage backend.
-2 Create bucket.
-3 Upload object.
-4 Download object.
-5 Migrate objects based on bucket across cloud.
-6 Create lifecycle for buckets.
+1. Register object storage backend.
```

Comment:
SODA Dashboard UI are not looking like sub-sections of the same? (looks same heading)
KIndly check

Reply:
I did not quite understand the comment. Could you please elaborate. 
Do we need to keep these sections: Volume Service, Fileshare Service and Multicloud Service.?
Our dashboard supports more features and this section will not be able to cover all.
Since this is an installation guide I feel we can remove the section.